gradle/wrapper/gradle-wrapper.properties

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+#Thu Aug 22 10:20:57 PDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
5.6 is already ok, right?

Reply:
You are right ;), just now I update the project with latest: https://services.gradle.org/distributions/